From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect enum usb_device_speed
Patch-mainline: never, kabi

In 3.12.64, commit 8488473e2db3f291a2d706f0dfc0848cbbb6901e (usb:
define USB_SPEED_SUPER_PLUS speed for SuperSpeedPlus USB3.1 devices),
upstream commit 8a1b2725a60d3267135c15e80984b4406054f650 added a value
to enum usb_device_speed. It confuses the kABI checker now.

Since it is only a new value in enum which changes nothing, we can
just protect the change the usual way, by #ifdef __GENKSYMS__.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/uapi/linux/usb/ch9.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/uapi/linux/usb/ch9.h
+++ b/include/uapi/linux/usb/ch9.h
@@ -942,7 +942,9 @@ enum usb_device_speed {
 	USB_SPEED_HIGH,				/* usb 2.0 */
 	USB_SPEED_WIRELESS,			/* wireless (usb 2.5) */
 	USB_SPEED_SUPER,			/* usb 3.0 */
+#ifndef __GENKSYMS__
 	USB_SPEED_SUPER_PLUS,			/* usb 3.1 */
+#endif
 };
 
 
